src/main/java/com/greenfoxacademy/goddesstribesbackend/models/dtos/UserIdAndKingdomIdDTO.java

```diff
@@ -0,0 +1,29 @@
+package com.greenfoxacademy.goddesstribesbackend.models.dtos;
+
+public class UserIdAndKingdomIdDTO {
```

Comment:
Why the ID's in the name? 
Is this supposed to be a linking table? 
Where will this be used?